server/middleware/paymentCardGuard.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const PaymentCard = require('../database/models/PaymentCard');
+
+module.export = function(req, res, next) {
+  PaymentCard.where({ user_id: req.user.id })
```

Comment:
sorry, I led you astray on this one.

It'll need to be:
PaymentCard.where({ id: req.params.id })

this is because this guard will only be used on the put/delete routes for payment cards.  On the get/post routes, we have built in security because the queries themselves rely on req.user.id